### ISSUE TITLE
[CI] WatcherUpgradeIT.testWatchesKeepRunning failing on 6.8

### ISSUE THREAD

**Issue by matriv** [elastic/elasticsearch#46929]
Logs: https://gradle-enterprise.elastic.co/s/ntqslfjcqxmzo/console-log?task=:x-pack:qa:rolling-upgrade:without-system-key:v6.0.0%23oneThirdUpgradedTestRunner

Cannot reproduce locally with:
```
./gradlew ':x-pack:qa:rolling-upgrade:without-system-key:v6.0.0#oneThirdUpgradedTestRunner' \
  -Dtests.seed=DDA15A625CD7B30E \
  -Dtests.class=org.elasticsearch.upgrades.WatcherUpgradeIT \
  -Dtests.method="testWatchesKeepRunning" \
  -Dtests.security.manager=true \
  -Dtests.locale=sq-AL \
  -Dtests.timezone=SystemV/YST9YDT \
  -Dtests.distribution=default \
  -Dcompiler.java=12 \
  -Druntime.java=8
```

Stacktrace:
```

FAILURE 33.0s \| WatcherUpgradeIT.testWatchesKeepRunning <<< FAILURES!
--
> Throwable #1: java.lang.AssertionError:
> Expected: a value greater than <187>
>      but: <187> was equal to <187>
> 	at __randomizedtesting.SeedInfo.seed([DDA15A625CD7B30E:40784CABD16676BD]:0)
> 	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
> 	at org.elasticsearch.upgrades.WatcherUpgradeIT.lambda$testWatchesKeepRunning$0(WatcherUpgradeIT.java:93)
> 	at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:884)
> 	at org.elasticsearch.upgrades.WatcherUpgradeIT.testWatchesKeepRunning(WatcherUpgradeIT.java:91)
> 	at java.lang.Thread.run(Thread.java:748)
```

Last occurrence almost one month ago.

**Comment by ywelsch** [comment]
More failures: https://gradle-enterprise.elastic.co/s/t7lgmsotkok3q

Muted the test on 6.8 in 60bb929ee3d

**Comment by jakelandis** [comment]
Closing this issue as won't fix since this has been open for a long time and there are no plans to invest the time necessary to fix this test. Multiple people have spent considerable time trying to get the Watcher tests stable in 7.x and master. Those changes include many updates to the test framework as well as a few production level changes that allow for better stop/start/stop/start in quick succession workflows (those only happen in the tests). The vast majority of test failures are due to the stop/start/stop nature and either leaving behind artifacts, or timing issues w.r.t stopping not actually being fully stopped before starting again. Since the identified issues in 7.x and master have not been real issues and the root cause of these failures is how we test, and that none of these changes to stabilize tests were back ported to 6.8 I will close this issue and leave the test muted. Additionally, this specific test is testing the upgrade from 5.x to 6.8 and there have not been any reports of issues with that upgrade and this test does not exist in 7.x+.
